content/docs/ibm/install-kubeflow.md

```diff
@@ -0,0 +1,220 @@
++++
+title = "Install Kubeflow"
+description = "Instructions for deploying Kubeflow with the shell"
+weight = 4
++++
+
+This guide describes how to use the kfctl binary to
+deploy Kubeflow on IBM Cloud.
+
+## Prerequisites
+
+### Installing the IBM Cloud developer tools
+
+> If you already have `ibmcloud` installed with the `ibmcloud cs` plugin, you
+> can skip these steps.
+
+1.  Download and install the `ibmcloud` command line tool:
+    https://console.bluemix.net/docs/cli/index.html#overview
+
+1.  Install the `cs` (container-service) plugin:
+    ```bash
+    ibmcloud plugin install container-service -r Bluemix
+    ```
+1.  Authorize `ibmcloud`:
```

Comment:
The numbered list is broken here. I think adding an empty line above this one will fix the problem. Alternatively, remove the `bash` language indicators - they often cause problems with our Markdown renderer.

See preview:
https://deploy-preview-1575--competent-brattain-de2d6d.netlify.com/docs/ibm/install-kubeflow/